ui/web/src/main/java/com/ttbmp/cinehub/ui/web/buyticket/ChooseSeatsViewController.java

```diff
@@ -0,0 +1,99 @@
+package com.ttbmp.cinehub.ui.web.buyticket;
+
+import com.ttbmp.cinehub.app.dto.ProjectionDto;
+import com.ttbmp.cinehub.app.dto.TicketDto;
+import com.ttbmp.cinehub.app.usecase.buyticket.BuyTicketUseCase;
+import com.ttbmp.cinehub.app.usecase.buyticket.Handler;
+import com.ttbmp.cinehub.app.usecase.buyticket.request.GetNumberOfSeatsRequest;
+import com.ttbmp.cinehub.app.usecase.buyticket.request.GetProjectionRequest;
+import com.ttbmp.cinehub.ui.web.domain.Projection;
+import com.ttbmp.cinehub.ui.web.domain.Seat;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Palmieri Ivan
+ */
+@Controller
+public class ChooseSeatsViewController {
+
+    private final String projectionList="projectionList";
+
+    @PostMapping("/choose_seat")
+    public String chooseSeats(@ModelAttribute("projection") Projection projection,
+            Model model) {
+
+        BuyTicketUseCase buyTicketUseCase = new Handler(new BuyTicketPresenterWeb(model));
+        buyTicketUseCase.getProjectionList(new GetProjectionRequest(
```

Comment:
The Projection class should be a superset of the ProjectionDto.
You must be able to create a ProjectionDto object from a Projection one on the fly without calling `buyTicketUseCase.getProjectionList`

Reply:
Impossible to create a projectionDto class directly from the projection class, as projection being a bean does not have all the attributes of the projectionDto class, for example instead of having a movieDto as an attribute, it has a movieId